src/business-logic/services/ingest-data-changed-service.ts

```diff
@@ -254,6 +277,7 @@ export class IngestDataChangedService implements DataChangeService {
     }
     const eventCallback: (() => Promise<void>) | undefined = this.getEventToExecute()
     if (!eventCallback) {
+      void this.generateActions()
```

Comment:
We should not use void. If generateAction fails, then the entire server will crash.
Instead add a `.catch` to the call and log the error. E.g.:
```typescript
this.generateActions().catch(error => this.logger.data(error).error('Failed generating actions for ingest batch.'))
```

Reply:
Updated